springql-core/src/api/low_level_rs.rs

```diff
@@ -4,201 +4,6 @@
 //!
 //! C API and high-level Rust API are provided separately.
 
-mod engine_mutex;
-mod spring_config;
+pub mod spring_config;
```

Comment:
We maybe able to totally remove this `low_level_rs` mod.

And then, we won't need to say "high-level" nor "low-level" anywhere.